src/app/styles/typography.scss

```diff
@@ -1,73 +1,81 @@
 @import './variables';
 
-$header-font-family: 'museo-sans', sans-serif;
-$default-font-family: 'Open Sans', sans-serif;
+@font-face {
+  font-family: 'museo';
+  src: url('./assets/fonts/museo/museo-300.otf') format("opentype");
+  font-weight: 300;
+}
 
-body {
-  h1 {
-    font-family: $header-font-family;
-    font-size: 42px;
-    font-weight: 700;
-  }
+@font-face {
+    font-family: 'museo';
+    src: url('./assets/fonts/museo/museo-500.otf') format("opentype");
+    font-weight: 500;
+}
 
-  h2 {
-    font-family: $header-font-family;
-    font-size: 36px;
+@font-face {
+    font-family: 'museo';
+    src: url('./assets/fonts/museo/museo-700.otf') format("opentype");
     font-weight: 700;
-  }
+}
 
-  h3 {
-    font-family: $header-font-family;
-    font-size: 24px;
-    font-weight: 700;
-  }
+// font stacks
+$museo:                "museo", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
I liked having `$header-font-family` defined,  since the intent is really clear. So perhaps adding `$header-font-family: $museo` would be a good idea?